src/Storage/Storage/Blob/Cmdlet/CopyAzureStorageBlob.cs

```diff
@@ -371,9 +397,137 @@ private async Task CopyFromUri(long taskId, IStorageBlobManagement destChannel,
 
                 switch (destBlobType)
                 {
-                    case Track2Models.BlobType.Block:
+                    case Track2Models.BlobType.Page:
+                        PageBlobClient destPageBlob = (PageBlobClient)Util.GetTrack2BlobClientWithType(destBlob, destChannel.StorageContext, Track2Models.BlobType.Page, ClientOptions);
+
+                        Track2Models.PageBlobCreateOptions pageBlobCreateOptions = new Track2Models.PageBlobCreateOptions();
+                        Track2Models.BlobHttpHeaders httpHeaders = new Track2Models.BlobHttpHeaders
```

Comment:
(Optional) better to make the prepare httpheader code to a function, to avoid redundant code of 3 blob type.

Reply:
Done. Moved the code block out of the switch statement.